server/src/main/java/org/opensearch/action/admin/cluster/stats/MappingVisitor.java

```diff
@@ -40,7 +40,7 @@
  *
  * @opensearch.internal
  */
-final class MappingVisitor {
+public final class MappingVisitor {
```

Comment:
Do you not need the `visitMapping` method to be public for using within the the plugin class. Also, since this class is annotated as `@opensearch.internal`, let us not make it public. We can duplicate the code within plugin if needed